src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine[] arrayMachine = {truck, bulldozer, excavator};
+        for (Machine help : arrayMachine) {
+            printInfo(help);
+        }
+    }
 
+    public static void printInfo(Machine machine) {
+        machine.doWork();
+        machine.stopWork();
+    }
```

Comment:
```suggestion
            machine.doWork();
            machine.stopWork();
        }
    }
```

I don't think we need separate method for this

Reply:
Can I keep foreach by fixing  array names and deleting unneeded method?